Thinktecture.Relay/Server/AcknowledgmentMode.cs

```diff
@@ -0,0 +1,24 @@
+namespace Thinktecture.Relay.Server
+{
+	public enum AcknowledgmentMode
+	{
+		/// <summary>
+		/// Requests in the message queue will be automatically acknowledged by
+		/// the OnPremiseConnector before executing the request against the on premise api
+		/// </summary>
+		Default,
+
+		/// <summary>
+		/// Requests in the message queue will be automatically acknowledged by
```

Comment:
`Requests in the message queue will be automatically acknowledged by the relay server before sending to the on-premise`